03_launch_mgmt_cluster.sh

```diff
@@ -185,22 +184,10 @@ function launch_baremetal_operator() {
     update_images
     popd
 
-    if [ "${BMO_RUN_LOCAL}" = true ]; then
-      touch bmo.out.log
-      touch bmo.err.log
-      kustomize build "$kustomize_overlay_path" | kubectl apply -f-
-      kubectl scale deployment metal3-baremetal-operator -n metal3 --replicas=0
-    fi
-
     if [ "${BMO_RUN_LOCAL}" = true ] || [ "${EPHEMERAL_CLUSTER}" = kind ]; then
       ${RUN_LOCAL_IRONIC_SCRIPT}
     fi
-
-    if [ "${BMO_RUN_LOCAL}" = true ]; then
-      nohup "${SCRIPTDIR}/hack/run-bmo-loop.sh" >> bmo.out.log 2>>bmo.err.log &
-    else
-      kustomize build "$kustomize_overlay_path" | kubectl apply -f-
-    fi
+    kustomize build "$kustomize_overlay_path" | kubectl apply -f-
```

Comment:
this needs to be in the else of the previous if (line 195 / 187)

Reply:
Done